### PR TITLE
make waxjs accept WAX purchase action

### PR DESCRIPTION
**Pull request by danielAlvess**

### CODE REVIEW

**Comment by porkchop** [comment]
This looks great @danielAlvess In addition I think we should also add the option to request the wax fallback fee as well. This way it will be backwards compatibvle with people using an old version of waxjs.
Like someowhere in this constructor object https://github.com/worldwide-asset-exchange/waxjs/blob/develop/src/index.ts#L45 we should probably add `feeFallback = true` by default. This way it will be automatically enabled when people upgrade

**Review by porkchop — APPROVED** [review]
LGTM @danielAlvess Nice1!

File: src/WaxSigningApi.ts [hunk 1]
```diff
@@ -59,11 +59,16 @@ export class WaxSigningApi {
   public async signing(
     transaction: Transaction,
     serializedTransaction: Uint8Array,
-    noModify = false
+    noModify = false,
+    feeFalback = true
```
Comment (porkchop) [inline src/WaxSigningApi.ts:63]: @danielAlvess we have a typo here that needs to be fixed to line up with the backend: feeFalback->feeFallback

**Review by porkchop — DISMISSED** [review]

**Comment by danielAlvess** [comment]
@porkchop I fixed typo and updated readme

**Review by niksad8 — APPROVED** [review]
let get this out there

**Pull request merged**